Makefile.toml

```diff
@@ -12,6 +12,7 @@ args = [
   "--workspace",
 ]
 command = "cargo"
+env = { RUST_MIN_STACK = "33554432" }
```

Comment:
`lookup_keccak` involving some deep recursion when build the circuit. I think it's a existing issue, just the new change hit the threshold unluckily. With that, we temporarily increase RUST_MIN_STACK to make tests pass. It doesn't break functional correctness.